README.md

```
@@ -2,119 +2,127 @@
 
-### Coverage Report
-
-You need to install lcov to generate report
-```bash
-# Mac OS
-brew install lcov
+## Structure
 
-# Linux
-apt-get install lcov
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)